pkg/network/driver/netlink/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
```

Comment:
correct me if I'm wrong here - but it looks like you moved code from the common.go file. no?

Reply:
I selectively copied from "common.go" in these first commits (the adapters).
In the last commit I removed the old ones.

Reply:
ACK. please specify that you are copying code from "common.go" on the commit desc, and also mention it will be deleted in a later commit

Reply:
Done